pkg/controllers/workgenerator/controller.go

```diff
@@ -1245,11 +1245,20 @@ func extractResFromConfigMap(uConfigMap *unstructured.Unstructured) ([]fleetv1be
 		return nil, err
 	}
 	// the list order is not stable as the map traverse is random
-	for _, value := range configMap.Data {
```

Comment:
not sure what happens when configMap.Data is empty.

Reply:
it's okey for now and there is a test for this case.